### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ColemanConnectIQ**
This change updates the readme with expanded info.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,77 @@
-# connectiq-apps
-A collection of Connect IQ apps and libraries.
+# /connectiq-apps
+_A collection of Connect IQ apps and libraries_
+
+------------------
+
+## Purpose
+This repo is officially maintained by the Garmin&reg; Connect IQ team. We are happy to have you here. It is our hope that this repository will extend the  resources available to our developers to help them build exciting new content!
```
Comment (BrandonConnectIQ) [inline README.md]: Double space in "extend the__resources available"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,77 @@
...
+
+## Other Useful Info
+
+### Forums:
+
+Our main form of communication is through the official [Connect IQ forums]. In our forums you have access to our our community of amazing developers as well as the ear of the Connect IQ team.
+
+The forum includes several spaces for our developer community:
+
+* [Connect IQ General Forum][Connect IQ forums]: This is the main forum and is the firs stop for help and developer discussions.
```
Comment (BrandonConnectIQ) [inline README.md]: firs -> first

File: README.md [hunk 3]
```diff
@@ -1,2 +1,77 @@
...
+------------------
+
+## Purpose
+This repo is officially maintained by the Garmin&reg; Connect IQ team. We are happy to have you here. It is our hope that this repository will extend the  resources available to our developers to help them build exciting new content!
+
+----------------------
+
+## What's in the Repo?
+
+This repo contains several resources for developers to help in the development process including examples and Garmin Monkey Barrel Code libraries:
```
Comment (BrandonConnectIQ) [inline README.md]: Should we say a little more along the lines of this being examples for reference, and not something that will be regularly augmented? I'm not sure... I just don't want developers to get the idea that we're going to be frequently growing a collection of example apps here.

**Review by BrandonConnectIQ — CHANGES_REQUESTED** [review]
Just a couple of minor things--looks really nice!

File: README.md (hunk 1 above)
Comment (ColemanConnectIQ) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (ColemanConnectIQ) [reply README.md]: Reworded just a little to denote a small set for reference.

File: README.md (hunk 2 above)
Comment (ColemanConnectIQ) [reply README.md]: Done

**Comment by ColemanConnectIQ** [comment]
Comments addressed.

**Review by BrandonConnectIQ — APPROVED** [review]

**Pull request merged**